Directory.Build.props

```diff
@@ -1,10 +1,68 @@
 <Project>
 
-  <Import Condition=" Exists('$(MSBuildThisFileDirectory)\..\Directory.Build.props') " Project="$(MSBuildThisFileDirectory)\..\Directory.Build.props" />
-  <Import Condition=" !Exists('$(MSBuildThisFileDirectory)\..\Directory.Build.props') " Project="$(MSBuildThisFileDirectory)\Directory.Build.Local.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <Import Project="$(RepositoryEngineeringDir)Analyzers.props" />
 
-  <PropertyGroup Condition=" Exists('$(MSBuildThisFileDirectory)\..\Directory.Build.props') ">
-    <PackageLicenseExpressionInternal></PackageLicenseExpressionInternal>
+  <PropertyGroup>
+    <IsSourceProject Condition="$([MSBuild]::MakeRelative($(RepoRoot), $(MSBuildProjectFullPath)).StartsWith('src/')) OR $([MSBuild]::MakeRelative($(RepoRoot), $(MSBuildProjectFullPath)).StartsWith('src\'))">true</IsSourceProject>
+  </PropertyGroup>
+
+  <!-- Language configuration -->
+  <PropertyGroup>
+    <!-- default to allowing all language features -->
+    <LangVersion>preview</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!--
+        Make sure any documentation comments which are included in code get checked for syntax during the build, but do
+        not report warnings for missing comments.
+
+        CS1573: Parameter 'parameter' has no matching param tag in the XML comment for 'parameter' (but other parameters do)
+        CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
+      -->
+    <NoWarn>$(NoWarn),1573,1591</NoWarn>
+  </PropertyGroup>
+
+  <!-- The TFMs to build and test against. -->
+  <PropertyGroup>
+    <NetFrameworkMinimum>net462</NetFrameworkMinimum>
+    <UwpMinimum>uap10.0.16299</UwpMinimum>
+    <WinUiMinimum>net6.0-windows10.0.18362.0</WinUiMinimum>
+    <NetCurrent>net8.0</NetCurrent>
+
+    <SupportedNetFrameworks>netcoreapp3.1;net6.0;net7.0;net8.0</SupportedNetFrameworks>
+    <MicrosoftTestingTargetFrameworks>net6.0;net7.0;net8.0</MicrosoftTestingTargetFrameworks>
+  </PropertyGroup>
+
+  <!-- Build config -->
+  <PropertyGroup>
+    <!-- Prevent warning about deprecated target frameworks -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
+    <!-- PDB -->
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>embedded</DebugType>
+    <!-- Workaround for AD0001 in analyzers with .NET 9. See https://github.com/dotnet/arcade/issues/14311 -->
+    <BuildWithNetFrameworkHostedCompiler>true</BuildWithNetFrameworkHostedCompiler>
+  </PropertyGroup>
+
+  <!-- Pack config -->
+  <PropertyGroup>
+    <Copyright>$(CopyrightMicrosoft)</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <DoNotShipTestingPlatformPackage>true</DoNotShipTestingPlatformPackage>
```

Comment:
No longer needed

Reply:
Are you going to delete it?
```suggestion
```

Reply:
Yes I just want to check locally if that's still used somewhere. GH is not working well when you have that many files.